spring-service-layer-validation/README.md

```diff
@@ -0,0 +1,12 @@
+## Spring Boot MVC
+
+This module contains articles about validation in Service layer of Spring Boot project.
+
+### Relevant Articles:
+
+- [Circular View Path Error](https://www.baeldung.com/spring-circular-view-path-error)
+- [Download an Image or a File with Spring MVC](https://www.baeldung.com/spring-controller-return-image-file)
+- [Spring MVC Async vs Spring WebFlux](https://www.baeldung.com/spring-mvc-async-vs-webflux)
+- [Differences in @Valid and @Validated Annotations in Spring](https://www.baeldung.com/spring-valid-vs-validated)
+- [CharacterEncodingFilter In SpringBoot](https://www.baeldung.com/spring-boot-characterencodingfilter)
+- More articles: [[prev -->]](/spring-boot-modules/spring-boot-mvc-2)
```

Comment:
I'm quite sure those are copy-pasted and not updated

Reply:
I have moved the code to spring-boot-modules and updated the README.md file.